docs/reference/search/request-body.asciidoc

```diff
@@ -188,46 +188,48 @@ to the client. This means it includes the time spent waiting in thread pools,
 executing a distributed search across the whole cluster and gathering all the
 results.
 
-include::request/query.asciidoc[]
```

Comment:
Thanks for the PR! I agree that these parameters need a reorganization.

Not blocking, but I wonder if there is some value is leaving basic or commonly used parameters, like `query` or `sort` near the top.

It seems like the pertinent question is "how are users currently accessing these parameter pages?" I don't currently have access to Google Analytics or web analytics data (working on it), but that might give us an indicator.

That said, I'm good with proceeding if others are.